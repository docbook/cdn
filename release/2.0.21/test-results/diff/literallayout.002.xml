<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <pre deltaxml:deltaV2="A" class="literallayout language-none line-numbers">
            <code>
               <a href="http://docbook.org/tdg/en/html/literallayout.html">LiteralLayout</a>: This is a test
    This is a test with <em>    embedded
  markup</em>; but still
  Only a test.
Had this been a real emegency,
  We would have fled in terror
And you would not have been informed.</code>
         </pre><div deltaxml:deltaV2="B" class="content">
            <pre class="literallayout language-none line-numbers">
               <code>
                  <a href="http://docbook.org/tdg/en/html/literallayout.html">LiteralLayout</a>: This is a test
    This is a test with <em>    embedded
  markup</em>; but still
  Only a test.
Had this been a real emegency,
  We would have fled in terror
And you would not have been informed.</code>
            </pre>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>