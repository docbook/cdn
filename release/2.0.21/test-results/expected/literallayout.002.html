<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><pre class="literallayout language-none line-numbers"><code><a href="http://docbook.org/tdg/en/html/literallayout.html">LiteralLayout</a>: This is a test
    This is a test with <em>    embedded
  markup</em>; but still
  Only a test.
Had this been a real emegency,
  We would have fled in terror
And you would not have been informed.</code></pre></article><script src="../../resources/js/prism.js"></script></body></html>